source/includes/connect/connection-options.php

```diff
@@ -0,0 +1,21 @@
+// start-connection-uri
+// Replace the placeholders with your actual hostname, port, and path to the certificate key file
+$uri = "mongodb://<hostname>:<port>/?tls=true&tlsCertificateKeyFile=/path/to/file.pem";
+
+// Create a MongoDB client
+$client = new MongoDB\Client($uri);
+// end-connection-uri
+
+// start-client-options
+// Replace the placeholders with your actual hostname and port
+$uri = "mongodb://<hostname>:<port>/";
+
+// Set the connection options
```

Comment:
Should this also include a comment to replace the path placeholder, as you suggestion in `start-connection-uri`?